### PR TITLE
wowTURKEY fix

### PR DESCRIPTION
**Pull request by erkinalp**
This removes the background images of the website and reply buttons. The background image of this website interferes with contrast setting, so needs to be removed.

### CODE REVIEW

**Comment by Gusted** [comment]
Where can I find `.cevapButton` and `span.postdetails span.name a`

**Comment by erkinalp** [comment]
You can find those in `viewtopic.php` aka thread page.

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -5915,6 +5915,26 @@ body {
...
 ================================
 
+wowturkey.com
+
+CSS
+body {
+    background: none !important;
+}
+.cevapButton {
+     background: var(--darkreader-neutral-background) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:5925]: 4 spaces

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -5915,6 +5915,26 @@ body {
...
+     background-image: none !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:5926]: 4 spaces

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -5915,6 +5915,26 @@ body {
...
+}
+span.postdetails span.name a {
+     background: transparent !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:5929]: 4 spaces

**Pull request merged**